Dockerfile

```
@@ -0,0 +1,5 @@
+FROM alpine:3.11
+
+COPY docs /docs
+
+CMD [ "exit", "-1" ]
```

Also, it would be nice to print something to the stdout so that it appears in the analysis status message

---

Something like: (maybe add a link to the repo with the documentation in the message)
```suggestion
RUN echo "echo 'clang-tidy cannot be run by Codacy' && exit 1" > /run.sh

ENTRYPOINT [ "sh", "/run.sh" ]

```

---

Thank you. I will :)

---

`sh` is already the alpine entrypoint :)

---

Still I would rather override the entrypoint anyways, since that way you make sure it doesn't change if you bump alpine versions

---

Done ✅